zos_concepts/gdg_datasets/copy_edit_fetch/README.md

```diff
@@ -0,0 +1,89 @@
+# Create, add, edit, copy and fetch with Generation data group
+This playbook demonstrates how to create, edit, copy and fetch
+Generation Data Groups (GDG) and Generation Datasets (GDS).
+
+This playbook uses:
+  - collection:
+    - ibm.ibm_zos_core
+  - modules:
+    - zos_copy
+    - zos_dataset
+    - zos_fetch
+    - zos_blockinfile
+    - zos_lineinfile
+
+It is a good practice to review the playbook contents before executing
+them. It will help you understand the requirements in terms of space, location,
+names, authority, and the artifacts that will be created and cleaned up.
+
+> [!IMPORTANT]
+> The release of `ibm_zos_core` collection version 1.10.0 introduced case-sensitive
+> changes to modules, specifically that module choices must be lower case. See the porting
+> guide in the release notes
+> ([here](https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html#porting-guide))
+> to understand the full scope of these changes.
+>
+> These changes are not compatible with earlier versions of the `ibm_zos_core` collection,
+> and since the earlier versions do not reach their end of life until April 2025,
+> the sample playbook compatible with the older versions will remain available.
+>
+> To support both the latest and prior versions of the collection:
+> * The `main` branch will reflect playbooks compatible with `ibm_zos_core` version 1.10.0 or later.
+>   * Find this playbook on the `main` branch: [here](https://github.com/IBM/z_ansible_collections_samples/tree/main/zos_concepts/data_sets/copy_edit_submit).
+> * The `ibm_zos_core-v1.9.x-samples` branch will reflect playbooks compatible with `ibm_zos_core` version 1.9.x or earlier.
+>   * Find this playbook on the `ibm_zos_core-v1.9.x-samples` branch: [here](https://github.com/IBM/z_ansible_collections_samples/tree/ibm_zos_core-v1.9.x-samples/zos_concepts/data_sets/copy_edit_submit).
+>
+> The `ibm_zos_core-v1.9.x-samples` branch will remain available until April 2025 but will not receive updates.
+>
+>
+
+## Playbook Requirements
+
+This playbook requires:
+- [IBM® z/OS® core collection 1.11.0 or later](https://galaxy.ansible.com/ibm/ibm_zos_core)
+- [Ansible® 2.5 or later](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
```

Comment:
It says here this playbook requires Ansible 2.5 or later but down in runtime-requirements, required versions are 2.15 and 2.16. Is this something common in other repository playbooks ?